engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -181,7 +194,7 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
           throw new IllegalArgumentException(s"Unexpected body part name: ${bodyPart.name.getOrElse("None")}")
         }
       })
-      partialSourcesToSourceCollections(partialSources, allowNoInputs)
+      eitherToTry(partialSourcesToSourceCollections(partialSources, allowNoInputs))
```

Comment:
try making this def `implicit` and see if the compiler complains.  Then you can delete this part